src/components/ProductCard/ProductCard.css

```diff
@@ -26,4 +27,13 @@
 .link{
     text-decoration: none;
     color: var(--white);
+}
+
+.wishlist-hover{
+    cursor: pointer;
+}
+
+.heartfull{
+    color: #ff4343;
+    
```

Comment:
use variables for colors

Reply:
yes in last will do all of that refactor work and make code clean.